apps/crn-server/src/handlers/teams/algolia-index-team-handler.ts

```diff
@@ -28,7 +28,10 @@ export const indexTeamHandler =
 
       if (team) {
         await algoliaClient.save({
-          data: addTagsFunction(team) as TeamDataObject,
+          data: {
+            ...addTagsFunction(team),
```

Comment:
Do we not need to save more data to algolia like the team name?

Reply:
actually we're saving more data than we need here 😅 these addTagsFunction returns all data + tags https://github.com/yldio/asap-hub/blob/861d4637a47fe17cc8341c8b94fdbdcbae6bd3ca/apps/crn-server/src/handlers/helper.ts#L7

Reply:
We might want to refactor this function to just return tags

Reply:
I've ended up removing this `addTagsFunction` from here, as we were sending more data than we need anyway. 
![Screenshot 2023-11-29 at 10 53 08](https://github.com/yldio/asap-hub/assets/16595804/c7eb6a8f-f99e-45bc-a0b9-60aa4ae18e4d)

Maybe down the road we can remove `addTagsFunction` everywhere. But now I don't think it makes sense to refactor this function to just return `_tags` as it is being used in several places to return data + _tags